vars/validateArtifacts.groovy

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/**
+SignArtifacts signs the given artifacts and saves the signature in the same directory.
```

Comment:
Description needs to be fixed!

Reply:
My bad, missed it while shifting between branches, will update it, thanks